lib/tasks/orangeindex.rake

```diff
@@ -168,10 +168,10 @@ namespace :liberate do
   end
 
   desc "Index a single MARC XML file against SET_URL"
-  task :index_file, [:file_name] => :environment do |_t, args|
+  task index_file: :environment do |_t, args|
```

Comment:
I think we don't need the block argument `|_t, args|` anymore now

Reply:
Fixed.